.travis.yml

```diff
@@ -29,33 +30,27 @@ matrix:
     - rvm: ruby-head
     - rvm: jruby-head
   exclude:
-    - rvm: 1.9.3
-      jdk: openjdk7
-      env: CI="travis"
-    - rvm: 1.9.3
-      jdk: oraclejdk8
-      env: CI="travis"
-    - rvm: 2.0.0
-      jdk: openjdk7
-      env: CI="travis"
-    - rvm: 2.0.0
-      jdk: oraclejdk8
-      env: CI="travis"
-    - rvm: 2.1.5
-      jdk: openjdk7
-      env: CI="travis"
-    - rvm: 2.1.5
-      jdk: oraclejdk8
-      env: CI="travis"
-    - rvm: 2.2.0
-      jdk: openjdk7
-      env: CI="travis"
-    - rvm: 2.2.0
-      jdk: oraclejdk8
-      env: CI="travis"
-    - rvm: ruby-head
-      jdk: openjdk7
-      env: CI="travis"
-    - rvm: ruby-head
-      jdk: oraclejdk8
-      env: CI="travis"
+    - rvm: jruby-9.1
+      env: MONGODB=2.6
+    - rvm: jruby-9.1
+      env: MONGODB=3.0
+    - rvm: jruby-9.1
+      env: MONGODB=3.2
+    - rvm: jruby-9.1
+      env: MONGODB=3.4
```

Comment:
Similarly, if we're not making any connections to the server, we probably just need to have a matrix with ruby versions.